.github/workflows/push-tests.yml

```diff
@@ -119,6 +171,71 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
         if: failure() && github.ref == 'refs/heads/main'
 
+  wait-for-shards:
+    name: Wait for shards
+    needs: run-e2e-tests # run after shards
+    runs-on: ubuntu-22.04
+    #if: success() # only run when all shards have passed
+    # store success output flag for ci job
+    outputs:
+      success: ${{ steps.setoutput.outputs.success }}
+    steps:
+      - id: setoutput
+        run: echo "{success}={true}" >> $GITHUB_OUTPUT
+
+  # https://github.com/myieye/web-languageforge/blob/develop/.github/workflows/e2e-tests.yml
+  e2e-test:
+    name: e2e-test
+    if: always()
+    runs-on: ubuntu-latest
+    needs: [run-e2e-tests, wait-for-shards]
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16.14.2"
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-duration-seconds: 1800
+
+      - name: Login to ECR
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ secrets.ECR_REPO }}
+
+      - name: Download blob reports from GitHub Actions Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: all-blob-reports
+          path: all-blob-reports
+
+      - uses: actions/checkout@v3
+      - name: Merge Playwright sharded reports
+        run: |
+          npx playwright --version
+          npx playwright merge-reports --reporter html ./all-blob-reports
```

Comment:
command not found for some reason

Reply:
Asking Playwright in Discord here: https://discord.com/channels/807756831384403968/1141146168299094107